control/controlclient/direct.go

```diff
@@ -1198,8 +1198,9 @@ func answerPing(logf logger.Logf, c *http.Client, pr *tailcfg.PingRequest, pinge
 		switch t {
 		case "TSMP", "disco":
 			go doPingerPing(logf, c, pr, pinger, t)
+		case "host":
+			go doPingerPing(logf, c, pr, &ICMPPinger{}, t)
```

Comment:
ICMPPinger{} unless you foresee this growing state or configuration over time?

Reply:
I don't, I want to actually replace the pinger abstraction here, but I didn't want to inline that refactor with this change.